game-core/src/main/java/games/strategy/triplea/util/UnitCategory.java

```diff
@@ -36,6 +37,7 @@
   private final BigDecimal movement;
   // movement of the units
   private final int transportCost;
+  private final boolean wasAmphibious;
```

Comment:
Why past tense with 'was'? A unit that is amphibious during combat, is actively an amphib assault, no? I feel like this might make more sense adding the word 'assault' in, as a unit can be amphibious (meaning they can attack from a transport), though 'conductedAmphibiouAssault' I think is the more exact (and verbose) way to describe it. The past tense of 'was' does strike me as a bit odd (for example, a casual reader may wonder when does this value become true, before a combat is initiated, during, or only after?).

Reply:
`wasAmphibious` is the term used in the `UnitAttachment` and so I was trying to keep the same terminology.  But since I used `amphibious` in the UnitSeparator, I've changed this to just `amphibious`.  See https://github.com/triplea-game/triplea/pull/8920/commits/c6950787c6db43e1aeebf8a31ead4830c7a7c4db